### PR TITLE
7187 expose appts physical location in its own field

### PR DESCRIPTION
**Pull request by cadibemma**
## Summary
- FE may want to display this field to help people find the exact location of their appts on large VA campuses. Right now we use it as a fallback for service_name, but we should add a new field for it also.

## Related issue(s)
- department-of-veterans-affairs/va-mobile-app#7187

## Testing done
- None

## What areas of the site does it impact?
- None

## Acceptance criteria

- [x]  New field for appts physical_location returned in response
- [x]  Specs updated
- [x]  Documentation has been updated

### CODE REVIEW

File: modules/mobile/docs/examples/appointments/va_appointment_request.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ data:
           extension: null
         url: null
         code: null
+      physicalLocation: null
```
Comment (kpethtel) [inline modules/mobile/docs/examples/appointments/va_appointment_request.yml:27]: Does physical location only apply to VA appointments and not CC and requests?

**Review by kpethtel — CHANGES_REQUESTED** [review]
Looks good aside from the redis caching issue.

File: modules/mobile/docs/examples/appointments/va_appointment_request.yml (hunk 1 above)
Comment (cadibemma) [reply modules/mobile/docs/examples/appointments/va_appointment_request.yml:27]: physical location only apply to VA appointments but I'm not sure if the would include va appointment requests.

File: modules/mobile/app/models/mobile/v0/appointment.rb [hunk 2]
```diff
@@ -56,6 +56,7 @@ class Appointment < Common::Resource
       attribute :healthcare_provider, Types::String.optional
       attribute :healthcare_service, Types::String.optional
       attribute :location, Location.optional
+      attribute :physical_location, Types::String.optional
```
[the remark this answers is not in the document]
Comment (cadibemma) [reply modules/mobile/app/models/mobile/v0/appointment.rb:59]: fixed

**Review by kpethtel — APPROVED** [review]
👍

**Pull request merged**